### DISCUSSION TITLE
What are the latent variables in FindMarkers function for?

### DISCUSSION THREAD

**Discussion by SirKuikka** [satijalab/seurat#4595] [General]
Hi,

I am a little confused about the latent.vars argument in Seurat's FindMarkers function.

latent.vars	
Variables to test, used only when test.use is one of 'LR', 'negbinom', 'poisson', or 'MAST'

So does using latent.vars give some additional benefit if I for example specify the individual in the metadata and use "individual" in latent.vars. Would this be a good idea if I have cells from different indivividuals and I compare two conditions within a same cell type.

**Comment by timoast** [comment]
The `latent.vars` argument specifies latent variables to include in the differential expression model. If you have data from different batches, for example, it's a good idea to include batch as a latent variable as this will essentially test whether the difference in gene expression between the groups can be explained by the difference in batch

**Reply by ThamrongC2** [reply]
Hi timoast

Please let me ask: In the context of 10x Genomics UMI-based single-cell RNA-seq data analyzed using MAST through Seurat’s FindMarkers framework, what is the recommended choice of latent variable for adjusting technical variation: nCount_RNA (total UMI counts per cell) or nFeature_RNA (number of detected genes per cell)?

Is there a preferred or recommended default between these two for UMI-based 10x data when using MAST, and are there any scenarios where one is clearly favored over the other from the perspective of controlling technical confounding without removing biological signal?
